core/src/test/java/org/elasticsearch/search/aggregations/AggregationsTests.java

```diff
@@ -93,7 +52,7 @@
     private static final List<InternalAggregationTestCase> aggsTests = getAggsTests();
 
     private static List<InternalAggregationTestCase> getAggsTests() {
-        List<InternalAggregationTestCase> aggsTests = new ArrayList<>();
+        List<InternalAggregationTestCase> aggsTests = new ArrayList<>();/*
```

Comment:
Absolutely... I ran a last test with on date histogram buckets with max number of buckets and sub aggregations and then forgot to remove the comments :(